README.md

```diff
@@ -72,31 +72,52 @@ const vc = require('vc-js');
 const suite = new Ed25519Signature2018({
   verificationMethod: publicKey.id,
   key: new Ed25519KeyPair({privateKeyBase58})
-})
+});
 
 vc.issue({ credential, suite })
     .then(issuedVc => console.log)
-    .catch(console.error)
+    .catch(console.error);
 ```
 
-### `verify()`
+### Creating a Verifiable Presentation - `createPresentation()`
+
+To create a Verifiable Presentation out of one or more Verifiable Credentials:
+
+```js
+const vc = require('vc-js');
```

Comment:
I don't think the `require` line needs to be in every snippet.  It makes it look like the snippet is perhaps an executable example, but these all require setup of other vars.

Reply:
Thanks, fixed.